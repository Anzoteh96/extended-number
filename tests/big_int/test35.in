-2578033124765186748742487846264580878894413610901844270810713103974898990660070004149729955295401294355683
-203576708841020377968595582051947005878483608832149329902962696857324476765949778522087964924738250672856
