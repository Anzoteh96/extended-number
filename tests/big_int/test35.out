Enter two numbers n and m:
n is -2578033124765186748742487846264580878894413610901844270810713103974898990660070004149729955295401294355683
m is -203576708841020377968595582051947005878483608832149329902962696857324476765949778522087964924738250672856
n + m is -2781609833606207126711083428316527884772897219733993600713675800832223467426019782671817920220139545028539
n - m is -2374456415924166370773892264212633873015930002069694940907750407117574513894120225627641990370663043682827
n * m is 524827498822828384319795706406500650832447661588533393326138690245113057693057319322993714472906297031073089402454327470165091619076864474304970214433444389882459914936410714229206766115554089863708289737440648
n / m is 12
n % m is -135112618672942213119340861641216808352610304916052311975160741687005269468672661884674376198542286281411
